src/wallet/coincontrol.h

```diff
@@ -12,6 +12,7 @@
 #include <script/keyorigin.h>
 #include <script/signingprovider.h>
 #include <script/standard.h>
+#include <unordered_set>
```

Comment:
This should go with the other std lib includes.

Reply:
Fixed, thanks.